.travis.yml

```diff
@@ -21,5 +21,5 @@ script:
   - sh -x ./node_modules/patternfly-eng-release/scripts/_build.sh -o
 
 after_success:
-  - grunt build
+  - grunt build:production
```

Comment:
+1 for not building twice.  We would the `:production` target for anything travis buiids.

Can we pull the build step out of the release plugin so it's better exposed?  IMO it should be an explicit step in the travis config.  Scripts executed by travis should be single purpose for optimal clarity.